### PR TITLE
[6.15 RFE] Prepare for SCA only deprecation

### PR DESCRIPTION
**Pull request by ColeHiggins2**
6.15 Feature automation: SAT-20201

This test is verifying that that Simple Content Access endpoints are deprecated and will be required
        for all organizations in Satellite 6.16

As of now, Stream Satellite is showing Katello 4.12 instead of Satellite 6.16. Keeping in Draft state until changes are made

### CODE REVIEW

**Comment by ColeHiggins2** [comment]
airgun: https://github.com/SatelliteQE/airgun/pull/1026/files

**Comment by synkd** [comment]
trigger: test-robottelo
pytest: tests/foreman/test_organization.py::test_positive_prepare_for_sca_only_deprecation
airgun: 1026

**Comment by synkd** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_organization.py::test_positive_prepare_for_sca_only_deprecation
airgun: 1026

File: tests/foreman/ui/test_organization.py [hunk 1]
```diff
@@ -362,3 +362,26 @@ def test_positive_product_view_organization_switch(session, module_org, module_p
...
+            {
+                'name': gen_string('alpha'),
+                'label': gen_string('alpha'),
+                'sca': False,
+            }
+        )
+        results = target_sat.execute('tail -100 /var/log/foreman/production.log').stdout
+    assert (
+        'Simple Content Access will be required for all organizations in Satellite 6.16' in results
+    )
```
Comment (jyejare) [inline tests/foreman/ui/test_organization.py:387]: Isnt this warning logged in the notification panel ? And just being silently sent into logs only ?

If yes, please verify the notification mssg as well, If not should we force DEVs to log that in notification as well ?

**Review by jyejare — CHANGES_REQUESTED** [review]

**Pull request closed without merging**